roman_numeral.js

```diff
@@ -0,0 +1,55 @@
+/* Convert the given number into a roman numeral.*/
+
+/**
+ * @param {int} num
+ * @return {array}
+ * depending on the spec, returns an array with num details
+ */
+function getNumDetails(num){
+  var n = num;
+  var arr = [];
+  var spec = [1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1];
+  for(var i = 0; i < spec.length; i++){
+    arr.push(parseInt(n / spec[i]));
+    n = n % spec[i];
+  }
+  return arr;
+}
+
+/**
+ * @param {string, int} romanLetter,numberOfTimes
+ * @return {string}
+ * returns a string containing `romanLetter` concatenated a `numberOfTimes`
+ */
+function concatRoman(romanLetter, numberOfTimes){
```

Comment:
This function is way more generic than this. Once you figure out what it really does you could rename it repeatString. Or [better](https://www.google.com/search?q=javascript+repeat+string)